ee/tables/dataflattentable/tables.go

```diff
@@ -37,7 +38,10 @@ type Table struct {
 	cmdGen   allowedcmd.AllowedCommand
 	execArgs []string
 
-	keyValueSeparator string
+	keyValueSeparator  string
```

Comment:
I'm a little unsure about adding these. I kinda feel like we should be moving the parsing to it's own thing. But I can't _quite_ explain it

Reply:
I think the `dataflattentable.NewExecAndParseTable` pattern is more modular, since it's not just a bunch of options/struct variables on the table. 

I could also believe this is good enough, and we can iterate later. Maybe

Reply:
Yeah I felt icky with adding 3 new elements, but I wasn't sure of a clean way handle the requirements of the `StringDelimitedLineFunc ` method without them, at least for now.

Reply:
Since I was adding this to the `string_delimited` package I tried to format it closely to the `KeyValue` methods.